experimental/displaynames/tests/tests.rs

```diff
@@ -0,0 +1,65 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use icu_displaynames::{DisplayNamesOptions, LocaleDisplayNamesFormatter};
+use icu_locid::locale;
+use icu_locid::Locale;
+use std::str::FromStr;
+
+#[test]
+fn test_concatenate() {
+    #[derive(Debug)]
+    struct TestCase<'a> {
+        pub input_1: &'a Locale,
+        pub expected: &'a str,
+    }
+    let cases = [
+        TestCase {
+            input_1: &locale!("de-CH"),
+            expected: "Swiss High German",
+        },
+        TestCase {
+            input_1: &locale!("zh_Hans"),
+            expected: "Simplified Chinese",
+        },
```

Comment:
Thought: one other case could be "es-Latn-419"

Reply:
Done